homeassistant/components/aussie_broadband/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""The Aussie Broadband integration."""
+from __future__ import annotations
+
+from aiohttp import ClientError
+from aussiebb.asyncio import AussieBB, AuthenticationException
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import CONF_SERVICES, DOMAIN
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Aussie Broadband from a config entry."""
+
+    # Login to the Aussie Broadband API and retrieve the current service list
+    try:
+        client = AussieBB(
+            entry.data[CONF_USERNAME],
+            entry.data[CONF_PASSWORD],
+            async_get_clientsession(hass),
+        )
```

Comment:
Move the client construction out of the try block

Reply:
This has now been implemented.